Firestore/Example/Tests/Integration/FSTStreamTests.m

```diff
@@ -38,17 +38,25 @@ - (void)writesFinishedWithError:(NSError *_Nullable)error;
  */
 @interface FSTStreamStatusDelegate : NSObject <FSTWatchStreamDelegate, FSTWriteStreamDelegate>
 
+- (instancetype)initFrom:(XCTestCase *)testCase
+              usingQueue:(FSTDispatchQueue *)dispatchQueue NS_DESIGNATED_INITIALIZER;
+- (instancetype)init NS_UNAVAILABLE;
+
 @property(nonatomic, readonly) NSMutableArray<NSString *> *states;
 @property(atomic, readwrite) BOOL invokeCallbacks;
 @property(nonatomic, weak) XCTestExpectation *expectation;
 @property(nonatomic, weak, readonly) FSTStream *stream;
+@property(nonatomic, weak, readonly) XCTestCase *testCase;
```

Comment:
nit: consider making your declaration order match your assignment order in the constructor, so it's easy to verify everything is assigned.

Reply:
Done